server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -619,6 +624,49 @@ export function configureWebSocketServices(
 				);
 			}
 
+			// Send signal to collaboration session from broadcast-signal endpoint
+			if (collborationSessionEventEmitter !== undefined) {
```

Comment:
nitpick: we could get rid of the if statement and do collborationSessionEventEmitter?.on(...)